src/unix/assets/86Box.spec

```
@@ -0,0 +1,123 @@
+# Fedora RPM spec file for 86Box including roms
+#
+# To create RPM files from this spec file, run the following commands:
+#  sudo dnf install rpm-build
+#  mkdir -p ~/rpmbuild/{BUILD,RPMS,SOURCES,SPECS,SRPMS}
+#
+# copy this 86Box.spec file to ~/rpmbuild/SPECS and run the following commands:
+#  cd ~/rpmbuild
+#  sudo dnf builddep SPECS/86Box.spec
+#  rpmbuild --undefine=_disable_source_fetch -ba SPECS/86Box.spec
+#
+# After a successful build, you can install the RPMs as follows:
+#  sudo dnf install RPMS/$(uname -m)/86Box-3* RPMS/noarch/86Box-roms*
+
+%global romver 20220319
+
+Name:		86Box
+Version:	3.3
+Release:	1%{?dist}
+Summary:	Classic PC emulator
+License:	GPLv2
+URL:		https://86box.net
+
+Source0:	https://github.com/86Box/86Box/archive/refs/tags/v%%{version}.tar.gz
+Source1:	https://github.com/86Box/roms/archive/refs/tags/%{romver}.tar.gz
+
+BuildRequires: cmake
+BuildRequires: desktop-file-utils
+BuildRequires: extra-cmake-modules
+BuildRequires: gcc-c++
+BuildRequires: libFAudio-devel
+BuildRequires: libappstream-glib
+BuildRequires: libevdev-devel
+BuildRequires: libXi-devel
+BuildRequires: ninja-build
+BuildRequires: qt5-linguist
+BuildRequires: qt5-qtconfiguration-devel
+BuildRequires: qt5-qtbase-private-devel
+BuildRequires: qt5-qtbase-static
+BuildRequires: rtmidi-devel
+BuildRequires: wayland-devel
+BuildRequires: SDL2-devel
+
+Requires: hicolor-icon-theme
+Requires: fluid-soundfont-gm
+Requires: 86Box-roms
+
+%description
+86Box is a hypervisor and IBM PC system emulator that specializes in
+running old operating systems and software designed for IBM
+PC systems and compatibles from 1981 through fairly recent
+system designs based on the PCI bus.
+
+It supports various models of PCs, graphics and sound cards, and CPUs.
+
+%package	roms
+Summary:	ROMs for use with 86Box
+Version:	%{romver}
+License:	Proprietary
+BuildArch:	noarch
+
+%description	roms
+Collection of ROMs for use with 86Box.
+
+%prep
+%autosetup -p1 -a1
+
+%build
+%ifarch i386 x86_64
+  %cmake -DRELEASE=on
+%else
+  %ifarch arm aarch64
+    %cmake -DRELEASE=on -DDYNAREC=off -DNEW_DYNAREC=on
```

```suggestion
    %cmake -DRELEASE=on -DNEW_DYNAREC=on
```
The `NEW_DYNAREC` variable by itself switches between the old and new dynarec. Setting `DYNAREC` to false actually toggles dynamic recompilation off altogether.